docs/sphinx/CMakeLists.txt

```diff
@@ -1,13 +1,14 @@
 set(SPHINX_BUILD_DIR "${CMAKE_CURRENT_BINARY_DIR}/_build")
 set(SPHINX_HTML_DIR "${CMAKE_CURRENT_BINARY_DIR}/html")
 
-configure_file("${CMAKE_CURRENT_SOURCE_DIR}/conf.py"
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/conf.py.in"
```

Comment:
I found that changing config.py to config.py.in throws off the sphinx-build system. I've been building the documentation locally via sphinx-build sphinx/ build/ .

Reply:
We can't use sphinx configuration files that are generated at configuration. ReadtheDocs doesn't like them. We have to hard-code all sphinx options.